scripts/build-meetings.js

```diff
@@ -0,0 +1,57 @@
+const { writeFileSync } = require('fs');
+const { resolve } = require('path');
+const { graphql } = require('@octokit/graphql');
+
+require('dotenv').config({
+  path: resolve(process.cwd(), '.env.local'),
+});
+
+module.exports = async function buildMeetings() {
+  try {
+    const eventIssues = await graphql(
+      `
+        query eventIssues($owner: String!, $repo: String!) {
+          repository(owner: $owner, name: $repo) {
+            issues(labels: ["meeting"], states: [OPEN], first: 10) {
```

Comment:
if you display just 2 I guess better just call or 2?

Reply:
Unfortunately I can't. I am deciding which two to keep based on the event date and time. I have to fetch all of them and then compare their dates.

Reply:
omg of course 🤦🏼